test/wasm2c/check-imports.txt

```diff
@@ -802,11 +802,15 @@ FUNC_TYPE_T(w2c_test_t0) = "\x07\x80\x96\x7a\x42\xf7\x3e\xe6\x70\x5c\x2f\xac\x83
 FUNC_TYPE_T(w2c_test_t1) = "\x72\xab\x00\xdf\x20\x3d\xce\xa1\xf2\x29\xc7\x9d\x13\x40\x7e\x98\xac\x7d\x41\x4a\x53\x2e\x42\x42\x61\x55\x2e\xaa\xeb\xbe\xc6\x35";
 FUNC_TYPE_T(w2c_test_t2) = "\x92\xfb\x6a\xdf\x49\x07\x0a\x83\xbe\x08\x02\x68\xcd\xf6\x95\x27\x4a\xc2\xf3\xe5\xe4\x7d\x29\x49\xe8\xed\x42\x92\x6a\x9d\xda\xf0";
 
+static u32 w2cw_w2c_test_f1(void *instance) {
```

Comment:
Can we drip the second `w2c_` prefix here?   `w2cw_test_f1` should suffice?   

Also, how about using a more meaningful prefix?  `wrapper_w2c_test_f1`?    `void_wrapper_w2c_test_f1`?   I don't think it really matters so if you prefer `w2cw` thats fine with me too.

Reply:
I don't believe it can conflict.  All user-controlled variables have prefixes, so we are free to use whatever prefix we like.

Reply:
alright, we'll change it to `wrap_`. also rebase it we guess.